src/PhpSpec/Formatter/JUnitFormatter.php

```diff
@@ -52,7 +52,7 @@ class JUnitFormatter extends BasicFormatter
         ExampleEvent::SKIPPED => 'skipped',
     );
 
-    public function __construct(PresenterInterface $presenter, IO $io, StatisticsCollector $stats)
+    public function __construct(Presenter $presenter, IO $io, StatisticsCollector $stats)
```

Comment:
We can detect usage of deprecated APIs by removing them and running the test suite ;-)